src/execwrappers.cpp

```diff
@@ -618,6 +618,19 @@ patchUserEnv(const char *env[], const char *filename)
   ldPreloadStr += getUpdatedLdPreload(filename, userPreloadStr.c_str());
 
   result.push_back(ldPreloadStr);
+
+  /*
+   * The path virtualization plugin tries to set the new prefix env var on
+   * restart. However, for some applications (e.g. bash), setenv() won't affect
+   * the envp area, possibly because the application has its private copy of
```

Comment:
Since we already update and patch other env variables here, it's easier to add one more rather than changing the whole model. For now, let's keep it as is.